tango_simlib/tests/test_tango_sim_generator.py

```diff
@@ -18,63 +18,66 @@
 
 from tango_simlib import tango_sim_generator
 from tango_simlib.tests import test_sim_test_interface
-from tango_simlib.utilities import helper_module, sim_xmi_parser, fandango_json_parser
+from tango_simlib.utilities import (helper_module, sim_xmi_parser,
+    fandango_json_parser, simdd_json_parser)
 from tango_simlib.utilities.testutils import ClassCleanupUnittestMixin
 
 MODULE_LOGGER = logging.getLogger(__name__)
 
 
-class test_TangoSimGenDeviceIntegration(ClassCleanupUnittestMixin, unittest.TestCase):
+class test_TangoSimGenDeviceIntegrationXMI(ClassCleanupUnittestMixin, unittest.TestCase):
 
     longMessage = True
+    server_name = 'weather_ds'
+    data_descr_file = 'Weather.xmi'
+    sim_file_parser = sim_xmi_parser.XmiParser()
```

Comment:
This is quite specific to the `sim_xmi_parser`. It should be generic and the subclasses should provide the `server_name`, `data_descr_file` and the `sim_file_parser` values.